apps/mobile-wallet/src/screens/Dashboard/DashboardScreen.tsx

```diff
@@ -122,19 +121,11 @@ const DashboardScreen = ({ navigation, ...props }: ScreenProps) => {
       refreshControl={<RefreshSpinner />}
       hasBottomBar
       verticalGap
-      screenTitle={walletName}
+      contrastedBg
       headerOptions={{
         headerRight: () => <HeaderButtons />,
-        headerLeft: () => <WalletSwitchButton />,
-        headerTitle: walletName
+        headerLeft: () => <WalletSwitchButton />
       }}
-      TitleSideComponent={
-        <NetworkBadgeContainer>
-          <Pressable onPress={() => setIsSwitchNetworkModalOpen(true)}>
-            <ActiveNetworkBadge />
-          </Pressable>
-        </NetworkBadgeContainer>
```

Comment:
Is this component now gone, or transfered somewhere else?

Reply:
It's gone